cassandra/cqlSchema.sql

```diff
@@ -1,4 +1,4 @@
-CREATE KEYSPACE IF NOT EXISTS lairbnbbooking WITH replication = {'class': 'SimpleStrategy', 'replication_factor' : 3};
+CREATE KEYSPACE IF NOT EXISTS lairbnbbooking WITH replication = {'class': 'SimpleStrategy', 'replication_factor' : 1};
```

Comment:
I'm legitimately curious why you downshifted replication factor here. Is it because it's just on one machine?

Reply:
Yes, I figured it's not needed with one machine.